src/components/molecules/TodoBottomPanel.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+import TaskCounter from "@/components/atoms/TaskCounter.vue";
+import FilterButton from "@/components/atoms/FilterButton.vue";
+import ClearCompletedButton from "@/components/atoms/ClearCompletedButton.vue";
+import { useStore } from "vuex";
+
+const store = useStore();
+
+function setFilter(filterName) {
+  store.commit("setFilter", filterName);
+}
+
+function deleteCompletedTasks() {
+  store.commit("removeAllCompleted");
+}
+</script>
+
+<template>
+  <div class="bottom-panel">
+    <TaskCounter />
+    <FilterButton
+      @click="setFilter('incomplete')"
+      label="Show incomplete"
+      filter-name="incomplete"
+    />
+    <FilterButton
+      @click="setFilter('completed')"
+      label="Show completed"
+      filter-name="completed"
+    />
+    <FilterButton
+      @click="setFilter('all')"
+      label="Show all"
+      filter-name="all"
+    />
```

Comment:
Its a bad approach yo have 3 similar elements like this

Reply:
i replaced it with common buttons, is that what you meant?